backend-docker/server/apiserver.go

```diff
@@ -126,6 +221,26 @@ func NewServer(cfg Config, docker dckr.Client) *Server {
 
 // Start starts a new http listener
 func (s *Server) Start() error {
+	// Populate the list of services
+	images, err := s.docker.ListImages()
+	if err != nil {
+		log.Println("list of docker images: ", err)
+	} else {
+		for _, img := range images {
+			serviceStore.setService(string(img.ID), img)
+		}
+	}
+
+	// Populate the list of jobs
```

Comment:
In general, I don't think it's possible to populate the list of jobs from running containers: you won't know what the input/output volumes are anyway. So we can start with an empty list of jobs and only care about the jobs started by the user in the current session.